docs/modules/servers/pages/postgres/configure/opensearch.adoc

```diff
@@ -0,0 +1,36 @@
+= Postgresql James Server &mdash; opensearch.properties
+:navtitle: opensearch.properties
+
+:sample-configuration-prefix-url: https://github.com/apache/james-project/blob/postgresql/server/apps/postgres-app/sample-configuration
+:pages-path: postgres
+:server-name: Postgresql James server
+include::partial$configure/opensearch.adoc[]
+
+== Search overrides
+
+*Search overrides* allow resolution of predefined search queries against alternative sources of data
+and allow bypassing OpenSearch. This is useful to handle most resynchronisation queries that
+are simple enough to be resolved against Postgres.
+
+Possible values are:
+
+- `org.apache.james.mailbox.postgres.search.AllSearchOverride` Some IMAP clients uses SEARCH ALL to fully list messages in
```

Comment:
I think search overrides could be in the partial too? If I ain't wrong the only differences are the packages names right?

org.apache.james.mailbox.{dbName}.search.AllSearchOverride

right? :)